modules/ROOT/pages/dashboards.adoc

```diff
@@ -24,6 +24,23 @@ Custom dashboards in Anypoint Monitoring can bring together important metrics an
 
 For example, you can compare live data with historic data to detect anomalies and expedite the troubleshooting process.
 
+[[no-data-scenarios]]
+== No Data Scenarios for Dashboards
```

Comment:
I also stumbled over a message being in label font rather than syntax font, but I think you did the correct thing based on our updated style standard. It's going to give me pause every now and then, because some elements belong to more than one category....

Reply:
```suggestion
== Causes of No Data Messages for Dashboards
```